### DISCUSSION TITLE
Umami Analytics

### DISCUSSION THREAD

**Discussion by momin-riyadh** [timlrx/tailwind-nextjs-starter-blog#901] [Q&A] [answered]
How can I configure umami analytics? I have used Vercel host and copied `.env.example` to `.env` and imported it after sign in to get variable details.

![Screenshot_1](https://github.com/timlrx/tailwind-nextjs-starter-blog/assets/8707777/7302020f-4529-454e-b645-46d93bc54c21)
I have found umami analytics code `<script defer src="https://us.umami.is/script.js" data-website-id="lorem ipsum dolkor sit amet"></script>`

Here I also update __next.config.js__

```javascript
const ContentSecurityPolicy = `
  default-src 'self';
  script-src 'self' 'unsafe-eval' 'unsafe-inline' giscus.app us.umami.is analytics.umami.is;
  style-src 'self' 'unsafe-inline';
  img-src * blob: data:;
  media-src *.s3.amazonaws.com;
  connect-src *;
  font-src 'self';
  frame-src giscus.app
`
```

**Comment by timlrx** [comment]
Have you added `src: 'https://us.umami.is/script.js'` in the [config file](https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/main/data/siteMetadata.js#L28-L33)?

**Reply by momin-riyadh** [reply, marked as answer]
Yes and successfully configured!
